client/src/components/Header.js

```diff
@@ -1,20 +1,57 @@
-import { NavLink } from "react-router-dom";
+import { useState } from "react";
+import { Link, NavLink } from "react-router-dom";
 import { v4 as uuidv4 } from "uuid";
+import logo from "../assets/Images/swing_icon_colored.png";
+import "./Header.css";
 
 export default function Header() {
+  const [toggleNavigationLinks, setToggleNavigationLinks] = useState(
+    "Header__navigation--links--off"
+  );
+
+  const [toggleWrapper, setToggleWrapper] = useState(
+    "Header__navigation__wrapper--off"
+  );
+
+  const [toggleButton, setToggleButton] = useState(
+    "Header__navigation--toggle--off"
+  );
+
   if (localStorage.getItem("userId") === null) {
     return localStorage.setItem("userId", JSON.stringify(uuidv4()));
   }
```

Comment:
Whay are you returning in here? This would make the component not render anything, or worse, it can crash! Take a look